test/testsymboldatabase.cpp

```diff
@@ -3209,6 +3210,18 @@ class TestSymbolDatabase : public TestFixture {
         ASSERT_EQUALS(2U, fredAType->classDef->linenr());
     }
 
+    void namespaces5() { // #13967
+        GET_SYMBOL_DB("namespace test {\n"
+                      "  template <int S>\n"
+                      "  struct Test { int x[S]; };\n"
+                      "  const Test<64> test;\n"
+                      "}\n");
+        const Variable *x = db->getVariableFromVarId(2U);
+        ASSERT_EQUALS("x", x->name());
```

Comment:
Hmm I feel this test is a bit weird. Variable ids are just unique numbers. If the variable "x" would get varid 1002 in the future nothing is wrong about that but this test will fail. But on the other hand we have lots of similar tests already in testsymboldatabase so you just followed the common pattern!
So we can keep it..